lib/src/beacon_settings.dart

```diff
@@ -32,6 +32,79 @@ abstract class Settings {
     return s;
   }
 
+  /// Creates a new [Setting] that manages a [int].
```

Comment:
I think it'd be nice to have these be siblings to `derivedSetting` since they're really just a passthrough to the normal API--is that possible? _Should_ be a drag and drop from what I'm seeing.

Small nit if we make that change: Can we chance `a [int]` to `an [int]` 🙃

Reply:
I originally had them as extensions but for some weird reason it fails to compile.
![Screenshot_20240328_201000](https://github.com/btrautmann/beacon_settings/assets/30532952/51d29025-566e-44d6-afa8-aad9b38401f9)

Reply:
<img width="550" alt="Screenshot 2024-03-28 at 21 16 51" src="https://github.com/btrautmann/beacon_settings/assets/8343465/0b9f3577-6ada-4b65-a321-6cf294b181b7">

That's odd, I have no issues 🤔

Reply:
I fixed it, silly mistake! I put them outside the block